gen3/bin/kube-setup-cedar-wrapper.sh

```diff
@@ -1,13 +1,68 @@
 source "${GEN3_HOME}/gen3/lib/utils.sh"
 gen3_load "gen3/lib/kube-setup-init"
 
+create_client_and_secret() {
+    local hostname=$(gen3 api hostname)
+    local client_name="cedar_ingest_client"
+    gen3_log_info "kube-setup-cedar-wrapper" "creating fence ${client_name} for $hostname"
+    # delete any existing fence cedar clients
+    g3kubectl exec -c fence $(gen3 pod fence) -- fence-create client-delete --client ${client_name} > /dev/null 2>&1
+    local secrets=$(g3kubectl exec -c fence $(gen3 pod fence) -- fence-create client-create --client ${client_name} --grant-types client_credentials | tail -1)
+    # secrets looks like ('CLIENT_ID', 'CLIENT_SECRET')
+    if [[ ! $secrets =~ (\'(.*)\', \'(.*)\') ]]; then
+        gen3_log_err "kube-setup-cedar-wrapper" "Failed generating ${client_name}: $secrets"
```

Comment:
just want to confirm this should not expose the creds into the log right? since the creds should be in that specific format

Reply:
I think the intent was to capture and echo error statements. That isn't working for me. With a bad `client-create` command the variable is only capturing INFO messages, not errors. It might be safer to not echo back the `secrets` variable on failure.